Add how-to bug-report (#179)
anbox-cloud/discourse-docs: howto/troubleshoot/collect-info.md

The guides in this section describe how to collect troubleshooting information

> from various parts of the Anbox Cloud stack. These guides should be used when
> facing or reporting issues with Anbox Cloud.

Reviewer: ```suggestion
from various parts of the Anbox Cloud stack. These guides should be used to
analyse or report issues with Anbox Cloud.
```
Reasoning for the change: "to face" and "to report" are so different that it is kinda confusing to see them combined like this. "Analyse" and "report" are a bit more similar.

Author: You're right, it's much nicer this way!
